src/generics/utils.js

```diff
@@ -232,7 +232,7 @@ function validateInput(input, validationData, modelName) {
 						}
 					}
 					break
-
+				case 'INTEGER':
 				case 'NUMBER':
 					console.log('Type of', typeof fieldValue)
```

Comment:
@nevil-mathew console log not require,  @nevil-mathew is this will break any existing flow. ?

Reply:
This line was already there, will remove.